ngui/ui/index.html

```diff
@@ -14,7 +16,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="/manifest.json" />
-    <title>Hystax OptScale</title>
+    <title>FinOps for Cloud</title>
```

Comment:
This one should be verified with MPT Design System. I am not sure if we shouldn't add something like FinOps for Cloud | SoftwareOne

Reply:
or {any title} | SoftwareOne FineOps for Cloud

Reply:
Here is a general CF page: https://softwareone.atlassian.net/wiki/spaces/mpt/pages/5383159897/Page+Titles+Convention+based+on+the+Reverse+Navigation+Path
 
And here is a TDR page: https://softwareone.atlassian.net/wiki/spaces/mpt/pages/5562991377/Page+Title+TDR

Reply:
I think we need to make a decision based on those 2 articles